.github/workflows/pr-diff.yml

```diff
@@ -22,7 +22,6 @@ jobs:
         run: |
           mkdir .diff
           npm ci
-          npx lerna bootstrap
```

Comment:
added in release branch as main still required the bootstrap command - but doesn't any more